package.json

```diff
@@ -1,51 +1,44 @@
 {
   "name": "editorconfig-checker",
-  "version": "4.0.2",
+  "version": "5.0.0",
   "description": "A tool to verify that your files are in harmony with your .editorconfig",
-  "main": "src/index.js",
-  "bin": {
-    "editorconfig-checker": "./dist/index.js",
-    "ec": "./dist/index.js"
-  },
-  "files": [
-    "LICENSE",
-    "README.md",
-    "dist/**/**/*.js"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/editorconfig-checker/editorconfig-checker.javascript.git"
-  },
   "keywords": [
     "lint",
     "linting",
     "clean-code"
   ],
-  "author": "Max Strübing <mxstrbng@gmail.com> (https://github.com/mstruebing) (https://twitter.com/mxstrbng)",
+  "repository": "https://github.com/editorconfig-checker/editorconfig-checker.javascript",
   "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/editorconfig-checker/editorconfig-checker.javascript/issues"
-  },
-  "homepage": "https://github.com/editorconfig-checker/editorconfig-checker.javascript#readme",
-  "xo": {
-    "envs": [
-      "jest"
-    ]
+  "author": "Max Strübing <mxstrbng@gmail.com> (https://github.com/mstruebing) (https://twitter.com/mxstrbng)",
+  "bin": {
+    "ec": "./dist/index.js",
+    "editorconfig-checker": "./dist/index.js"
   },
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "prepare": "make build"
-  },
-  "dependencies": {
-    "https-proxy-agent": "^5.0.0",
-    "node-fetch": "^2.6.0",
-    "tar": "^6.0.0"
+    "lint": "eslint . --fix",
+    "prepublishOnly": "npm start",
+    "start": "rimraf dist && ncc build src/index.ts",
+    "test": "ts-node src"
   },
   "devDependencies": {
-    "@types/node": "^16.0.0",
-    "@types/node-fetch": "^2.5.5",
-    "prettier": "^2.0.1",
-    "tslint": "^6.0.0",
-    "tslint-config-prettier": "^1.18.0",
-    "typescript": "^4.0.2"
+    "@octokit/rest": "^18.12.0",
+    "@types/node": "^17.0.15",
+    "@types/node-fetch": "^2.5.12",
+    "@types/tar": "^6.1.1",
+    "@types/tmp": "^0.2.3",
+    "@typescript-eslint/eslint-plugin": "^5.10.2",
+    "@typescript-eslint/parser": "^5.10.2",
+    "@vercel/ncc": "^0.33.1",
```

Comment:
Is `ncc` responsible for having some dependencies which are clearly runtime dependencies i.e. `@octokit/rest` as development dependencies?

Reply:
ncc is compile a Node.js project into a single file.

it compiled bundle file include all dependencies.